library/core/src/main/java/com/google/android/exoplayer2/util/FlacStreamMetadata.java

```diff
@@ -35,7 +36,7 @@
   public final int channels;
   public final int bitsPerSample;
   public final long totalSamples;
-  @Nullable public final Metadata vorbisComments;
+  @Nullable public final Metadata flacMetadata;
```

Comment:
Maybe just call this `metadata` at this point.

Reply:
Done